main/EDInitMod.F90

```diff
@@ -293,11 +301,40 @@ subroutine set_site_properties( nsites, sites,bc_in )
           sites(s)%NF         = 0.0_r8         
           sites(s)%frac_burnt = 0.0_r8
          
-         ! PLACEHOLDER FOR PFT AREA DATA MOVED ACROSS INTERFACE                                                                                   
-          if(hlm_use_fixed_biogeog.eq.itrue)then
+
+          if(hlm_use_fixed_biogeog.eq.itrue)then  
+           ! MAPPING OF FATES PFTs on to HLM_PFTs
+            ! add up the area associated with each FATES PFT
+            ! where pft_areafrac is the area of land in each HLM PFT and (from surface dataset)
+            ! hlm_pft_map is the area of that land in each FATES PFT (from param file)
```

Comment:
just to be clear, are the logical constraints of hlm_pft_map that all rows and all columns need to sum to one? if so, can we add a check after parameter reading to ensure that that condition is satisfied?

Reply:
Just the one way round, so that for each HLM_PFT the values need to add up to one. I think in principle a FATES_PFT could accept >1 or <1 quantities of HLM_PFT area. I've added a check in 
https://github.com/rosiealice/fates/commit/9eefda82636e318b9594b175bee79d10eaff757f